### PR TITLE
 Add integration tests for oauth2 endpoints for legacy URL mode

### PR DESCRIPTION
**Pull request by GANGANI**
**Description**
This PR will add integration tests for oauth2 endpoints for legacy URL mode. 
- Add tests for the oauth2 backchannel logout endpoint and oauth2 JWKS endpoint for both tenant and super tenant legacy URL mode.
- Add tests for the Oauth2 userinfo, introspection, revocation, checksession endpoints for both tenant and super tenant legacy URL mode.
- Add tests for authorization code flow, password grant flow, client credential grant flow for tenant legacy URL mode.

Resolves : https://github.com/wso2/product-is/issues/8524

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.BasicCookieStore;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import org.testng.Assert;
+import org.testng.annotations.*;
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:33]: remove * annotations

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+            appDTO.setOAuthVersion(OAuth2Constant.OAUTH_VERSION_2);
+            appDTO.setGrantTypes(OAuth2Constant.OAUTH2_GRANT_TYPE_AUTHORIZATION_CODE);
+            appDTO.setBackChannelLogoutUrl("http://localhost:" + DEFAULT_TOMCAT_PORT + "/playground2/bclogout");
+
+            adminClient.registerOAuthApplicationData(appDTO);
+            OAuthConsumerAppDTO createdApp = adminClient.getOAuthAppByName(OIDC_APP_NAME);
+            Assert.assertNotNull(createdApp, "Adding OIDC app failed.");
+            oidcAppClientId = createdApp.getOauthConsumerKey();
+        } catch (Exception e) {
+            Assert.fail("Adding OIDC app failed.", e);
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:132]: do we need to assert failure here? we can add throw Exception to method signature.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+            response = sendGetRequest(locationHeader.getValue());
+            sessionDataKey = Utils.extractDataFromResponse(response, CommonConstants.SESSION_DATA_KEY, 1);
+            if (isCheckLogoutConfirmation) {
+                Assert.assertNotNull(sessionDataKey, "Back channel logout failed for OIDC.");
+            } else {
+                Assert.assertNotNull(sessionDataKey, "sessionDataKey is null for ." + OIDC_APP_NAME);
+            }
+            EntityUtils.consume(response.getEntity());
+        } catch (Exception e) {
+            Assert.fail("OIDC initiation request failed.", e);
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:161]: remove this and add 'throws Exception' to method signature

Comment (emswbandara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:161]: please check other places also

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java [hunk 4]
```diff
@@ -81,17 +84,34 @@
...
-        super.init(TestUserMode.SUPER_TENANT_USER);
 
         logManger = new AuthenticatorClient(backendURL);
         adminUsername = userInfo.getUserName();
         adminPassword = userInfo.getPassword();
-        String sessionIndex = logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-                isServer.getSuperTenant().getTenantAdmin().getPassword(),
-                isServer.getInstance().getHosts().get("default"));
+        String sessionIndex =
+                logManger.login(username, userPassword,	isServer.getInstance().getHosts().get("default"));
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java:114]: remove unnecessary spaces.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantTestCase.java [hunk 5]
```diff
@@ -63,21 +65,38 @@
...
     public void testInit() throws Exception {
 
-        super.init(TestUserMode.SUPER_TENANT_USER);
         logManger = new AuthenticatorClient(backendURL);
         adminUsername = userInfo.getUserName();
         adminPassword = userInfo.getPassword();
-        logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-                isServer.getSuperTenant().getTenantAdmin().getPassword(),
-                isServer.getInstance().getHosts().get("default"));
+        logManger.login(username, userPassword,	isServer.getInstance().getHosts().get("default"));
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantTestCase.java:99]: remove unnecessary spaces

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java [hunk 6]
```diff
@@ -47,19 +50,35 @@
...
 	private String consumerSecret;
+	private String username;
+	private String userPassword;
 
 	private CloseableHttpClient client;
 
+	@DataProvider(name = "configProvider")
+	public static Object[][] configProvider() {
+		return new Object[][]{
+				{TestUserMode.SUPER_TENANT_ADMIN},
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java:61]: i think line number 61 and 62 needs to be formatted. please check

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java [hunk 7]
```diff
@@ -58,16 +60,33 @@
 	private String consumerSecret;
 
 	private CloseableHttpClient client;
+	private final String username;
+	private final String userPassword;
+
+	@DataProvider(name = "configProvider")
+	public static Object[][] configProvider() {
+		return new Object[][]{
+				{TestUserMode.SUPER_TENANT_ADMIN},
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java:69]: format this line and below

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceIntrospectionTestCase.java [hunk 8]
```diff
@@ -50,17 +53,37 @@
...
     public void testInit() throws Exception {
 
-        super.init(TestUserMode.SUPER_TENANT_USER);
         logManger = new AuthenticatorClient(backendURL);
         adminUsername = userInfo.getUserName();
         adminPassword = userInfo.getPassword();
-        logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-                isServer.getSuperTenant().getTenantAdmin().getPassword(),
-                isServer.getInstance().getHosts().get("default"));
+        logManger.login(username, userPassword,	isServer.getInstance().getHosts().get("default"));
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceIntrospectionTestCase.java:86]: remove spaces

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java [hunk 9]
```diff
@@ -65,16 +68,36 @@
...
 	protected IdentityGovernanceServiceClient identityGovernanceServiceClient;
+	private final String username;
+	private final String userPassword;
+	private final String activeTenant;
+	private final String TENANT_DOMAIN = "wso2.com";
+
+	@DataProvider(name = "configProvider")
+	public static Object[][] configProvider() {
+		return new Object[][]{
+				{TestUserMode.SUPER_TENANT_ADMIN},
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java:79]: reformat lines

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java [hunk 10]
```diff
@@ -65,16 +68,36 @@
 	private static final String lockedUserPassword = "test_locked_user_pass";
 	private static final String ACCOUNT_LOCK_CLAIM_URI = "http://wso2.org/claims/identity/accountLocked";
 	protected IdentityGovernanceServiceClient identityGovernanceServiceClient;
+	private final String username;
+	private final String userPassword;
+	private final String activeTenant;
+	private final String TENANT_DOMAIN = "wso2.com";
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java:74]: let's define constant as static

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithRevokedAccessToken.java [hunk 11]
```diff
@@ -53,11 +55,20 @@
     private ClientID consumerKey;
     private Secret consumerSecret;
 
-    private String tokenType;
+    private final String tokenType;
+    private final String username;
+    private final String userPassword;
+    private final String activeTenant;
+    private final String TENANT_DOMAIN = "wso2.com";
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithRevokedAccessToken.java:62]: use static when defining constants

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 1 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java]: Fixed https://github.com/wso2/product-is/pull/8500/commits/947bac08aa1f82c91d337ca6d537cb8264debfa9

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 2 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 3 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantTestCase.java (hunk 5 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java (hunk 4 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java (hunk 7 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java (hunk 6 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceIntrospectionTestCase.java (hunk 8 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceIntrospectionTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java (hunk 10 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithRevokedAccessToken.java (hunk 11 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithRevokedAccessToken.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java (hunk 9 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java]: Fixed 947bac0

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java [hunk 12]
```diff
@@ -58,16 +60,30 @@
...
 	public void testInit() throws Exception {
-		super.init(TestUserMode.SUPER_TENANT_USER);
+
 		logManger = new AuthenticatorClient(backendURL);
 		adminUsername = userInfo.getUserName();
 		adminPassword = userInfo.getPassword();
-		logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-				isServer.getSuperTenant().getTenantAdmin().getPassword(),
-				isServer.getInstance().getHosts().get("default"));
+		logManger.login(username, userPassword,	isServer.getInstance().getHosts().get("default"));
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java:86]: change this line to 
        sessionCookie = logManger.login(username, userPassword, isServer.getInstance().getHosts().get("default"));

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceIntrospectionTestCase.java [hunk 13]
```diff
@@ -50,17 +53,33 @@
...
     public void testInit() throws Exception {
 
-        super.init(TestUserMode.SUPER_TENANT_USER);
         logManger = new AuthenticatorClient(backendURL);
         adminUsername = userInfo.getUserName();
         adminPassword = userInfo.getPassword();
-        logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-                isServer.getSuperTenant().getTenantAdmin().getPassword(),
-                isServer.getInstance().getHosts().get("default"));
+        logManger.login(username, userPassword, isServer.getInstance().getHosts().get("default"));
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceIntrospectionTestCase.java:82]: change to 
 sessionCookie = logManger.login(username, userPassword, isServer.getInstance().getHosts().get("default"));

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java [hunk 14]
```diff
@@ -65,16 +68,33 @@
...
 	public void testInit() throws Exception {
-		super.init(TestUserMode.SUPER_TENANT_USER);
+
 		logManger = new AuthenticatorClient(backendURL);
 		adminUsername = userInfo.getUserName();
 		adminPassword = userInfo.getPassword();
-		logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-				isServer.getSuperTenant().getTenantAdmin().getPassword(),
-				isServer.getInstance().getHosts().get("default"));
+		logManger.login(username, userPassword,	isServer.getInstance().getHosts().get("default"));
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceResourceOwnerTestCase.java:97]: change to  sessionCookie = logManger.login(username, userPassword, isServer.getInstance().getHosts().get("default"));

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java [hunk 15]
```diff
@@ -359,6 +370,7 @@ public HttpResponse sendLoginPost(HttpClient client, String sessionDataKey) thro
         return response;
     }
 
+
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java]: remove unnecessary new line

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java [hunk 16]
```diff
@@ -369,4 +381,5 @@ public HttpResponse sendLoginPost(HttpClient client, String sessionDataKey) thro
 
         return claimValues;
     }
-}
+
```
Comment (emswbandara) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java]: remove unnecessary new line

Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java]: Fixed https://github.com/wso2/product-is/pull/8500/commits/f3c8c97719e2480642ca3e50a14c54fbb87ab2ce

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java (hunk 15 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java]: Fixed https://github.com/wso2/product-is/pull/8500/commits/f3c8c97719e2480642ca3e50a14c54fbb87ab2ce

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 17]
```diff
@@ -0,0 +1,269 @@
...
+        Header locationHeader = response.getFirstHeader(OAuth2Constant.HTTP_RESPONSE_HEADER_LOCATION);
+        Assert.assertNotNull(locationHeader, "Authorization response header is null.");
+        EntityUtils.consume(response.getEntity());
+
+        response = sendGetRequest(locationHeader.getValue());
+        sessionDataKey = Utils.extractDataFromResponse(response, CommonConstants.SESSION_DATA_KEY, 1);
+        if (isCheckLogoutConfirmation) {
+            Assert.assertNotNull(sessionDataKey, "Back channel logout failed for OIDC.");
+        } else {
+            Assert.assertNotNull(sessionDataKey, "sessionDataKey is null for ." + OIDC_APP_NAME);
```
Comment (pamodaaw) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:155]: Incorrect formatting of the error message

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java [hunk 18]
```diff
@@ -81,22 +86,45 @@
...
+        adminClient = new OauthAdminClient(backendURL, sessionCookie);
+        remoteUSMServiceClient = new RemoteUserStoreManagerServiceClient(backendURL, sessionCookie);
 
         logManger = new AuthenticatorClient(backendURL);
-        adminUsername = userInfo.getUserName();
-        adminPassword = userInfo.getPassword();
-        String sessionIndex = logManger.login(isServer.getSuperTenant().getTenantAdmin().getUserName(),
-                isServer.getSuperTenant().getTenantAdmin().getPassword(),
-                isServer.getInstance().getHosts().get("default"));
+        String sessionIndex = logManger.login(username, userPassword,
```
Comment (pamodaaw) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java:120]: format this line

Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java:120]: This line is no longer there in this class.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 17 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:155]: Fixed https://github.com/wso2/product-is/pull/9453/commits/836990572286f04de75d9ce9c940e23502842139

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 19]
```diff
@@ -0,0 +1,269 @@
...
+import java.io.IOException;
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.wso2.identity.integration.test.utils.CommonConstants.DEFAULT_TOMCAT_PORT;
+
+public class OAuth2BackChannelLogoutTestCase extends OAuth2ServiceAbstractIntegrationTest {
```
Comment (janakamarasena) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:57]: Add class comment

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2JWKSEndpointTestCase.java [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.identity.integration.test.utils.DataExtractUtil;
+
+import java.io.IOException;
+
+public class Oauth2JWKSEndpointTestCase {
```
Comment (janakamarasena) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2JWKSEndpointTestCase.java:33]: Add class comment

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 19 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:57]: Fixed https://github.com/wso2/product-is/pull/9453/commits/e0df5f258fc6ecbd1f7b18b1c2bb53b37d7694d8

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2JWKSEndpointTestCase.java (hunk 20 above)
Comment (GANGANI) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2JWKSEndpointTestCase.java:33]: Fixed https://github.com/wso2/product-is/pull/9453/commits/e0df5f258fc6ecbd1f7b18b1c2bb53b37d7694d8

**Pull request merged**